exchanges/binanceus/binanceus_wrapper.go

```diff
@@ -0,0 +1,976 @@
+			orderType = order.Market
+		}
+
+		return order.Detail{
+			Amount:         resp.OrigQty,
+			Exchange:       bi.Name,
+			OrderID:        strconv.FormatInt(int64(resp.OrderID), 10),
+			ClientOrderID:  resp.ClientOrderID,
+			Side:           orderSide,
+			Type:           orderType,
+			Pair:           pair,
+			Cost:           resp.CummulativeQuoteQty,
+			AssetType:      assetType,
+			Status:         status,
+			Price:          resp.Price,
+			ExecutedAmount: resp.ExecutedQty,
+			Date:           resp.Time,
+			LastUpdated:    resp.UpdateTime,
+		}, nil
+	}
+	return respData, fmt.Errorf("%s %w", assetType, asset.ErrNotSupported)
+}
+
+// GetDepositAddress returns a deposit address for a specified currency
+func (bi *Binanceus) GetDepositAddress(ctx context.Context, c currency.Code, accountID, chain string) (*deposit.Address, error) {
+	address, err := bi.GetDepositAddressForCurrency(ctx, c.String(), chain)
+	if err != nil {
+		return nil, err
+	}
+	return &deposit.Address{
+		Address: address.Address,
+		Tag:     address.Tag,
+	}, nil
+}
+
+// WithdrawCryptocurrencyFunds returns a withdrawal ID when a withdrawal is
+// submitted
+func (bi *Binanceus) WithdrawCryptocurrencyFunds(ctx context.Context, withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	if err := withdrawRequest.Validate(); err != nil {
+		return nil, err
+	}
+	resp, er := bi.WithdrawCrypto(ctx, withdrawRequest, "", 0)
+	if er != nil {
+		return nil, er
+	}
+	return &withdraw.ExchangeResponse{
+		ID: resp,
+	}, nil
+}
+
+// WithdrawFiatFunds returns a withdrawal ID when a withdrawal is
+// submitted
+// But, GCT has no concept of withdrawal via SEN
+// the fiat withdrawal end point of Binance.US is built to submit a USD withdraw request via Silvergate Exchange Network (SEN).
+// So, this method is not implemented.
+func (bi *Binanceus) WithdrawFiatFunds(ctx context.Context, withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	return nil, common.ErrNotYetImplemented
+}
+
+// WithdrawFiatFundsToInternationalBank returns a withdrawal ID when a withdrawal is submitted
+// But, GCT has no concept of withdrawal via SEN
+// the fiat withdrawal end point of Binance.US is built to submit a USD withdraw request via Silvergate Exchange Network (SEN).
+func (bi *Binanceus) WithdrawFiatFundsToInternationalBank(ctx context.Context, withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	return nil, common.ErrNotYetImplemented
+}
+
+// GetActiveOrders retrieves any orders that are active/open
+func (bi *Binanceus) GetActiveOrders(ctx context.Context, getOrdersRequest *order.GetOrdersRequest) ([]order.Detail, error) {
+	if err := getOrdersRequest.Validate(); err != nil {
+		return nil, err
+	}
+	if len(getOrdersRequest.Pairs) == 0 || len(getOrdersRequest.Pairs) >= 40 {
+		getOrdersRequest.Pairs = append(getOrdersRequest.Pairs, currency.EMPTYPAIR)
+	}
+	var orders []order.Detail
+	if getOrdersRequest.AssetType == asset.Spot {
```

Comment:
If you invert this `if` statement to `if getOrdersRequest.AssetType != asset.Spot {` you can exit earlier and not indent the main piece of code